WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModelProtocol.swift

```diff
@@ -52,6 +52,8 @@ protocol ProductFormViewModelProtocol {
                              taxStatus: ProductTaxStatus,
                              taxClass: TaxClass?)
 
+    func updateReviews(averageRating: String, ratingCount: Int)
```

Comment:
Not sure at 100% about this, since if we want to use the same screen of Product Reviews, it will be possible to moderate them, and in theory, the count of `averageRating` and `ratingCount` will change.

Reply:
Like above 👍